app/views/works/_work_form_tags.html.erb

```diff
@@ -1,9 +1,18 @@
 <fieldset class="work meta">
   <legend><%= ts('Tags') %></legend>
   <h3 class="landmark heading"><%= ts('Tags') %></h3>
+
   <p class="note"><%= ts('Tags are comma separated, %{max} characters per tag.', :max => ArchiveConfig.TAG_MAX) %></p>
 
   <dl>
+    <% import ||= false
```

Comment:
I think the notes were much better off in their own fieldset -- they don't make much sense in a fieldset labeled Tags that has tag-specific instructions.